### PR TITLE
fix: Attendee, event and ticket entity conversion from dbflow to room

### LINKED ISSUE #1348: Replace the current database with Room
### Chore

**Description**
We should start using Room database (AAC) in the project.

**Would you like to work on the issue?**
No

### PR DESCRIPTION
**Pull request by tavishjain**
Partially Fixes #1348 

Checklist:

- [ ] I have checked for PMD and check-style issues 
- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream development branch.

Changes: [Add here what changes were made in this issue and if possible provide links.]

Screenshots for the change:

### CODE REVIEW

File: app/src/main/java/com/eventyay/organizer/data/attendee/Attendee.java [hunk 1]
```diff
@@ -43,88 +44,118 @@
...
+
+    @Override
+    public int getLayoutRes() {
+        return 0;
+    }
+
+    @Override
+    public int compareTo(@NonNull Attendee attendee) {
+        return 0;
+    }
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/data/attendee/Attendee.java]: These shouldn't be required

Comment (tavishjain) [reply app/src/main/java/com/eventyay/organizer/data/attendee/Attendee.java]: @iamareebjamal As the class implements Comparable, Android Studio is asking to implement these functions mandatorly

Comment (iamareebjamal) [reply app/src/main/java/com/eventyay/organizer/data/attendee/Attendee.java]: It did not ask before so how come is it now?

Comment (iamareebjamal) [reply app/src/main/java/com/eventyay/organizer/data/attendee/Attendee.java]: And your implementations are completely wrong, so it'll only cause bugs

I don't think you have followed README and applied the kombok plugin

Comment (tavishjain) [reply app/src/main/java/com/eventyay/organizer/data/attendee/Attendee.java]: When I changed @Table of DBFlow to @Entity of Room, this implementation error arised.

Comment (tavishjain) [reply app/src/main/java/com/eventyay/organizer/data/attendee/Attendee.java]: I'll have a look on how can I remove them, if they aren't necessary

Comment (iamareebjamal) [reply app/src/main/java/com/eventyay/organizer/data/attendee/Attendee.java]: An annotation change of Table to Entity can't cause this

**Comment by mariobehling** [comment]
A few months have passed since this PR. Unfortunately there was no follow up anymore. We will try to find someone else who can resolve this as the work on this PR seems to have stopped. Closing it therefore due to inactivity.

**Pull request closed without merging**